lib/active_model_serializers/lookup_chain.rb

```diff
@@ -0,0 +1,43 @@
+module ActiveModelSerializers
+  module LookupChain
+    BY_RESOURCE = lambda do |_resource_class, resource_class_name, _serializer_class|
+      "#{resource_class_name}Serializer"
+    end
+
+    BY_NAMESPACE = lambda  do |resource_class, resource_class_name, _serializer_class|
+      resource_namespace = resource_class.name.deconstantize
+      serializer_name = "#{resource_class_name}Serializer"
```

Comment:
@NullVoxPopuli @richmolj a class without state should just be a module using either `module_function` or `extend self` depending on if there's a chance you might want to include it (like file_utils) or just style